Require libssl-dev?

While testing a manual compile on a vivid64 virtualbox image (see #53 and #54) failed with:

```
scripts/selinux/genheaders/genheaders.c:129:18: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
    for (k = 0; k < max(1, 40 - strlen(map->name) - strlen(map->perms[j])); k++)
                  ^
  HOSTCC  scripts/selinux/mdp/mdp
  HOSTCC  scripts/kallsyms
  HOSTCC  scripts/conmakehash
  HOSTCC  scripts/sortextable
  HOSTCC  scripts/asn1_compiler
  HOSTCC  scripts/sign-file
scripts/sign-file.c:23:30: fatal error: openssl/opensslv.h: No such file or directory
 #include <openssl/opensslv.h>
                              ^
compilation terminated.
scripts/Makefile.host:126: recipe for target 'scripts/sign-file' failed
make[2]: *** [scripts/sign-file] Error 1
Makefile:547: recipe for target 'scripts' failed
make[1]: *** [scripts] Error 2
make[1]: Leaving directory '/home/vagrant/linux/linux-4.4.5'
debian/ruleset/targets/common.mk:295: recipe for target 'debian/stamp/build/kernel' failed
make: *** [debian/stamp/build/kernel] Error 2
```

This fixed it:

```
sudo aptitude install libssl-dev
```
Need to test more platforms to determine whether this is a vivid-specific issue. Tests and role validation tasks should identify supported platforms (#19, #33).

--- a/roles/build-grsec-kernel/vars/Ubuntu.yml
+++ b/roles/build-grsec-kernel/vars/Ubuntu.yml
@@ -7,5 +7,6 @@ apt_packages:
   - git-core
   - kernel-package
   - libncurses5-dev
+  - libssl-dev
   - make
   - python-requests

